schemas/type/decimal.md

```diff
@@ -8,28 +8,28 @@ point representation in order to avoid precision errors.
 A `Decimal` represents a decimal number in a form similar to scientific
 notation.
 
-It has two fields:
+It has two properties:
```

Comment:
I believe the reason these were described as "field" is because that's what protobuf calls them.

I think we had a discussion no what we would call the protocol-agnostic conflict, and landed on "fields" to match the existing AEP text. @rofrankel can correct me though.

Reply:
I don't remember the discussion, but at a first take that conclusion sounds good to my protobuf-biased self.

Aside from protobuf using one term and OAS using the other, is there an agnostic argument either way?

Reply:
To be clear, the `properties` term is not OAS specific but rather JSON Schema derived.   Here's a nice long summary from SO.  IMO, because we discussing a schema definition, I would prefer the use of `properties` but I'm open to discussion on it. 

https://stackoverflow.com/questions/295104/what-is-the-difference-between-a-field-and-a-property

Reply:
That's a fair clarification...though from the perspective of trying to be agnostic to protobuf vs. OAS, OAS and JSON Schema are effectively the same entity (they're always and only used together with respect to protobuf being an alternative).

Reply:
I'm ok either way.